app/code/Magento/CatalogGraphQl/Model/Resolver/Product/Websites.php

```diff
@@ -20,44 +18,30 @@
  */
 class Websites implements ResolverInterface
 {
-    /**
-     * @var ValueFactory
-     */
-    private $valueFactory;
-
     /**
      * @var Collection
      */
     private $productWebsitesCollection;
 
     /**
-     * @param ValueFactory $valueFactory
      * @param Collection $productWebsitesCollection
      */
     public function __construct(
-        ValueFactory $valueFactory,
         Collection $productWebsitesCollection
     ) {
-        $this->valueFactory = $valueFactory;
         $this->productWebsitesCollection = $productWebsitesCollection;
     }
 
     /**
-     * {@inheritDoc}
+     * @inheritdoc
      */
-    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null) : Value
+    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
     {
         if (!isset($value['entity_id'])) {
-            $result = function () {
-                return null;
-            };
-            return $this->valueFactory->create($result);
+            throw new GraphQlInputException(__('"model" value should be specified'));
         }
         $this->productWebsitesCollection->addIdFilters((int)$value['entity_id']);
```

Comment:
This was not supposed to be replaced because this was actual performance optimization via Deferred mechanism.